scene/theme/theme_db.cpp

```diff
@@ -381,25 +383,30 @@ void ThemeDB::get_class_own_items(const StringName &p_class_name, List<ThemeItem
 
 	HashSet<StringName> inherited_props;
 	for (const StringName &theme_type : class_hierarchy) {
-		HashMap<StringName, HashMap<StringName, ThemeItemBind>>::Iterator E = theme_item_binds.find(theme_type);
+		HashMap<StringName, List<ThemeItemBind>>::Iterator E = theme_item_binds_list.find(theme_type);
 		if (E) {
-			for (const KeyValue<StringName, ThemeItemBind> &F : E->value) {
-				if (inherited_props.has(F.value.item_name)) {
+			for (const ThemeItemBind &F : E->value) {
+				if (inherited_props.has(F.item_name)) {
 					continue; // Skip inherited properties.
 				}
-				if (F.value.external || F.value.class_name != p_class_name) {
-					inherited_props.insert(F.value.item_name);
+				if (F.external || F.class_name != p_class_name) {
+					inherited_props.insert(F.item_name);
 					continue; // Track properties defined in parent classes, and skip them.
 				}
 
-				r_list->push_back(F.value);
+				r_list->push_back(F);
 			}
 		}
 	}
 }
 
-// Object methods.
+void ThemeDB::_sort_theme_items() {
+	for (KeyValue<StringName, List<ThemeDB::ThemeItemBind>> &E : theme_item_binds_list) {
+		E.value.sort_custom<ThemeItemBind::SortByType>();
+	}
+}
 
+// Object methods.
```

Comment:
There was an empty line here, btw, to mark a section. It's not a comment for this specific method 🙃